packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -119,17 +119,53 @@ class _ToolbarContainerLayout extends SingleChildLayoutDelegate {
 ///
 /// An app bar consists of a toolbar and potentially other widgets, such as a
 /// [TabBar] and a [FlexibleSpaceBar]. App bars typically expose one or more
-/// common actions with [IconButton]s which are optionally followed by a
+/// common [actions] with [IconButton]s which are optionally followed by a
 /// [PopupMenuButton] for less common operations.
 ///
 /// App bars are typically used in the [Scaffold.appBar] property, which places
 /// the app bar as a fixed-height widget at the top of the screen. For a
 /// scrollable app bar, see [SliverAppBar], which embeds an [AppBar] in a sliver
 /// for use in a [CustomScrollView].
 ///
-/// The AppBar displays the toolbar widgets, [leading], [title], and
-/// [actions], above the [bottom] (if any). If a [flexibleSpace] widget is
-/// specified then it is stacked behind the toolbar and the bottom widget.
+/// The AppBar displays the toolbar widgets, [leading], [title], and [actions],
+/// above the [bottom] (if any). If a [flexibleSpace] widget is specified then
```

Comment:
I have trouble visualizing what purpose/common use pattern of the bottom is for just reading this and the next paragraphs. Maybe add a half sentence elaboration.